server/src/test/java/org/elasticsearch/index/mapper/vectors/DenseVectorFieldTypeTests.java

```diff
@@ -230,6 +230,7 @@ public void testCreateNestedKnnQuery() {
                 VectorData.fromFloats(queryVector),
```

Comment:
we need tests here that exercise the expected values (e.g. we set it, the query should have that value set). 

These tests would need to be guarded by the assumption that the feature flag is enabled.